src/components/avatars/OcAvatar.spec.js

```diff
@@ -0,0 +1,107 @@
+import { mount } from "@vue/test-utils"
+import OcAvatar from "./OcAvatar.vue"
+
+describe("OcAvatar", () => {
+  const selectors = {
+    initials: ".avatarInitials",
+  }
+  function getWrapperWithProps(props) {
+    return mount(OcAvatar, {
+      propsData: {
+        ...props,
+      },
+    })
+  }
+  describe("prop value", () => {
+    describe("when src is set", () => {
+      let wrapper
+      beforeEach(() => {
+        wrapper = getWrapperWithProps({
+          src: "http://some-image.jpg",
+        })
+      })
+      it("should render oc image", () => {
+        const imageElement = wrapper.find("img")
+        expect(imageElement.exists()).toBeTruthy()
+        expect(imageElement.attributes("src")).toBe("http://some-image.jpg")
+      })
+      it("should not render user initial", () => {
+        expect(wrapper.find(selectors.initials).exists()).toBeFalsy()
+      })
+      it("should not render background", () => {
+        expect(wrapper.attributes("style")).not.toContain("background-color: ")
+      })
+    })
+    describe("when username is set", () => {
+      it.each`
+        username                   | expected_initial
+        ${"."}                     | ${"."}
+        ${"ॐ 卐"}                  | ${"ॐ卐"}
```

Comment:
I know that it has different meaning for you, but please avoid having a svastika. 😅

Reply:
removed ;) !